package.json

```diff
@@ -38,6 +38,7 @@
     "starlight-blog": "^0.13.0",
     "starlight-links-validator": "^0.12.3",
     "starlight-showcases": "^0.2.0",
+    "starlight-sidebar-topics": "^0.1.0",
```

Comment:
Let's update this PR by bumping `starlight-sidebar-topics` to the newest release: 0.2.0.